tests-cuda/test_1276-cupy-interop.py

```diff
@@ -8,8 +8,8 @@
 
 
 def test_cupy_interop():
-    cupy = ak._nplikes.Cupy.instance()
-    numpy = ak._nplikes.Numpy.instance()
+    cupy = ak._nplikes.cupy.Cupy.instance()
+    numpy = ak._nplikes.numpy.Numpy.instance()
```

Comment:
In order to avoid breaking import cycles, the `_nplikes` package doesn't import `cupy` or `numpy`. Are you happy to import these at the top level, e.g. `from awkward._nplikes.cupy import Cupy`?

Reply:
Thanks for suggestion!